dataset/src/main/scala/frameless/TypedDataset.scala

```diff
@@ -212,24 +211,24 @@ class TypedDataset[T] protected[frameless](val dataset: Dataset[T])(implicit val
     */
   def apply[A](column: Witness.Lt[Symbol])
     (implicit
-      i0: TypedColumn.Exists[T, column.T, A],
-      i1: TypedEncoder[A]
+      i1: TypedColumn.Exists[T, column.T, A],
+      i2: TypedEncoder[A]
     ): TypedColumn[T, A] = col(column)
 
   /** Returns `TypedColumn` of type `A` given its name.
     *
     * {{{
     * tf.col('id)
     * }}}
-    *
-    * It is statically checked that column with such name exists and has type `A`.
+  *
+  * It is statically checked that column with such name exists and has type `A`.
     */
   def col[A](column: Witness.Lt[Symbol])
     (implicit
-      i0: TypedColumn.Exists[T, column.T, A],
-      i1: TypedEncoder[A]
+      i1: TypedColumn.Exists[T, column.T, A],
+      i2: TypedEncoder[A]
     ): TypedColumn[T, A] =
-      new TypedColumn[T, A](dataset(column.value.name).as[A](TypedExpressionEncoder[A]))
+    new TypedColumn[T, A](dataset(column.value.name).as[A](TypedExpressionEncoder[A]))
```

Comment:
I think the original indentation makes sense: +1 level compared to the equal sign :P